### ISSUE TITLE
warning when installing for dev

### ISSUE THREAD

**Issue by edouardoyallon** [kymatio/kymatio#649]
python setup.py develop

```
/Users/oyallon/anaconda/envs/kymatio/lib/python3.8/site-packages/setuptools/dist.py:461: UserWarning: Normalizing '0.2.a0' to '0.2a0'
  warnings.warn(tmpl.format(**locals()))
```

**Comment by janden** [comment]
Yeah I keep seeing this in the test logs. Not sure what it means? Versioning experts? @lostanlen @eickenberg?

**Comment by lostanlen** [comment]
I plead guilty:
https://github.com/kymatio/kymatio/commit/d081b892b0d86922a591fe98d7353d61e95d66d4#diff-45e8d3a5d6f1bfc72c4d9b1cd29e72ac07de13e8e7d71346c8451a72e6ef7e14

Looks like i misunderstood PEP 440 
https://www.python.org/dev/peps/pep-0440/

This shouldn't affect the behavior of the program but i don't think we can fix this warning unless we rewrite history

**Comment by edouardoyallon** [comment]
any thoughts?

**Comment by lostanlen** [comment]
the `version` of master should be `0.3.dev0` at this point

**Comment by edouardoyallon** [comment]
so what do we do about this

**Comment by lostanlen** [comment]
we need to change the version name and use setuptools to make sure that it passes the PEP440 specification
https://www.python.org/dev/peps/pep-0440/

**Comment by lostanlen** [comment]
closed by #700
